### PR TITLE
added a new servlet class

### PR DESCRIPTION
**Pull request by archthegit**
Nothing new, just following instructions on the codeU website.

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -53,22 +53,42 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/codeu/data/Datastore.java]: This is a bit of a nit but do you mind running autoformat on your code once and getting rid of the unnecessary new lines? :)

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -53,22 +53,42 @@ public void storeMessage(Message message) {
...
     Query query =
-        new Query("Message")
-            .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
-            .addSort("timestamp", SortDirection.DESCENDING);
+            new Query("Message")
+                    .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
+                    .addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
+
+    messages = createList(results, user);
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/codeu/data/Datastore.java]: You can `return createList(results, user);` directly and get rid of the local `messages` variable here.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 3]
```diff
@@ -53,22 +53,42 @@ public void storeMessage(Message message) {
...
+
+  }
+
+  public List<Message> getAllMessages(){
+    List<Message> messages;
+
+    Query query = new Query("Message").addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
 
+    messages = createList(results,null);
```
Comment (SuyashLakhotia) [inline src/main/java/com/google/codeu/data/Datastore.java]: Same thing here.

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (SuyashLakhotia) [reply src/main/java/com/google/codeu/data/Datastore.java]: ^ For both files.

Comment (archthegit) [reply src/main/java/com/google/codeu/data/Datastore.java]: I am not sure how exactly this is done on intelliJ, but I just pressed command+option+L. Hope it worked!

**Pull request merged**